### ISSUE TITLE
Adding bank transfer as a payment method

### ISSUE THREAD

**Issue by codePau** [remjx/coindrop#852]
Hi!
Would it be possible to add bank transfer as an option for receiving payments? Thanks and congratulations for the tool!

**Comment by remjx** [comment]
Yes, by the end of the year I plan to add the ability for users to add custom payment methods. Sorry for the inconvenience.

**Comment by edwincitox** [comment]
Hi @remjx 

Add a small form to be able to enter the personalized information to make international bank transfers.

The minimum required fields could be:
- Name of the bank
- SWIFT code, IBAN or ABA
- Bank address
- Name of the beneficiary
- Beneficiary account number
- Country, city and address of the beneficiary
